packages/gestalt/src/icons/shopping-bag.svg

```diff
@@ -1 +1 @@
-<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 5a2.5 2.5 0 0 1 5 0v2h-5V5zM17 7V5c0-2.76-2.24-5-5-5S7 2.24 7 5v2H2v15c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V7h-5z"/></svg>
+<svg height="24" width="24" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" d="M6 6L3 6C1.89543 6 1 6.89543 1 8V21C1 22.6569 2.34315 24 4 24H20C21.6569 24 23 22.6569 23 21V8C23 6.89543 22.1046 6 21 6H18C18 2.68629 15.3137 0 12 0C8.68629 0 6 2.68629 6 6ZM15 6C15 4.34315 13.6569 3 12 3C10.3431 3 9 4.34315 9 6L15 6ZM9 10.5C9 11.3284 8.32843 12 7.5 12C6.67157 12 6 11.3284 6 10.5C6 9.67157 6.67157 9 7.5 9C8.32843 9 9 9.67157 9 10.5ZM16.5 12C17.3284 12 18 11.3284 18 10.5C18 9.67157 17.3284 9 16.5 9C15.6716 9 15 9.67157 15 10.5C15 11.3284 15.6716 12 16.5 12Z"/></svg>
```

Comment:
@brunolucena 
![image](https://user-images.githubusercontent.com/10593890/112201406-972d4f00-8be6-11eb-9ed3-8e3585c073fe.png)

The d attribute is incorrect,  must be redone without `fill-rule="evenodd"`